### DISCUSSION TITLE
Repair Page Continues to Scan Even After Navigating Away

### DISCUSSION THREAD

**Discussion by RaulGw4** [immich-app/immich#12128] [Feature Request]
### The bug

When a user navigates to the "Repair" page on Immich, the application automatically begins scanning for files with issues. If the user navigates away from the Repair page to another page (e.g., Dashboard, Albums), the scanning process continues to run on the server. This can lead to significant resource usage, particularly when a large number of assets are being scanned. The HDD may run at full capacity for 15-20 minutes, which could negatively impact server performance.

### The OS that Immich Server is running on

Truenas Scale

### Version of Immich Server

v1.112.1

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Default for Truenas Apps
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

1. Click on the repair page;
2. Exit by going back to the main page;
3. Analyze the HDD usage.

**Comment by mertalev** [comment]
Once the server receives a request, it will process it regardless. I don't think this is really a bug, although it's of course better if it can stop processing.

The real issue IMO is that the repair page is too slow. If it could do an offline scan and present the results to you quickly, it wouldn't really matter if it doesn't cancel that request.
